### DISCUSSION TITLE
Where is Whisper installed? Where are the models installed?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [openai/whisper#1774] [Q&A] [answered]
I went into my WHisper folder to check where are the models located, and I was in shock to see that there was nothing inside that folder (the one where I did the whisper) except my videos and my transcriptions.
So... where is whisper being isntalled exactly? Same question for the models etc

**Comment by jongwook** [comment, marked as answer]
It places the model files under inside `.cache/whisper` under your home directory by default:

https://github.com/openai/whisper/blob/fcfeaf1b61994c071bba62da47d7846933576ac9/whisper/__init__.py#L128-L130

You can delete these files if your disk space is running out, and then the script will re-download them when you run Whisper next time.

**Reply by AIhasArrived** [reply]
Ah okay thank you, what about all the others files, and WHY are the files put on such obscure folder? Where are the other files downloaded? Is there any particular useful reason it must be placed there?, thankss

**Reply by jongwook** [reply]
It's quite common to store cached files in `~/.cache` to be reused from different applications and directories. Maybe on windows we could have used `%LOCALAPPDATA%`. We don't need to download and cache any other files than these model checkpoints.

**Comment by themanyone** [comment]
I found a bunch more models in `.cache/huggingface/hub/` from playing around with whisper python import.
`pipeline = FlaxWhisperPipline("openai/whisper-small.en")`
